app/views/moderation/inbox/index.haml

```diff
@@ -0,0 +1,25 @@
+- provide(:title, generate_title(t(".title", user: @user.screen_name)))
+
+.card.question--fixed
+  .container
+    .card-body
+      .media
+        %a.pull-left{ href: show_user_profile_path(@user.screen_name) }
+          %img.answerbox__question-user-avatar.avatar-md{ src: @user.profile_picture.url(:medium) }
+        .media-body
+          Viewing inbox for
+          %br
+          %strong= @user.screen_name
+          = " (##{@user.id})"
```

Comment:
One inbox entry is hidden behind this header.

You need to add duplicated markup to push down the container, see https://github.com/Retrospring/retrospring/blob/main/app/views/question/show.haml#L2-L3

Reply:
e024d8b